kalite/distributed/static/js/distributed/user/views.js

```diff
@@ -299,15 +312,25 @@ window.UserView = BaseView.extend({
                 this.totalPointViewXs = new TotalPointView({model: this.model, el: "#points-xs"});
             }
         } else {
+            var options = {};
+            var next = getParamValue("next");
+            var login = getParamValue("login");
+            if (login || this.login_start_open) {
+                options.start_open = true;
+
+                delete this.login_start_open;
```

Comment:
Some comments in here (and a few more, in general) could be helpful -- not clear to me what this variable (`login_start_open`) means, or why it's being deleted rather than set to `false`, etc.